apps/desktop/src/routes/+layout.svelte

```diff
@@ -92,13 +92,13 @@
 	});
 
 	const handleKeyDown = createKeybind({
-		'$mod+Equal': () => {
```

Comment:
I guess, we should use `+` instead of `=` for this. At least on German keyboards, they are on different keys (which was the reason for using `Equal` here).

Reply:
Hmm yeah so that seems to break the shortcut for English language keyboard users because then it technically has to be `$mod+Shift++` :thinking:

Reply:
How about just having both shortcuts? I can't immediately see anythign wrong with that :thinking: We're not using `$mod+=` for anything else

```ts
'$mod++': () => {
	zoom = Math.min(zoom + 0.0625, 3);
},
'$mod+=': () => {
	zoom = Math.min(zoom + 0.0625, 3);
},
```

Reply:
Ah - I wasn't aware of the issues in the English keyboard. To tackle that, we could use the feature to use regular expressions in the shortcuts. E.g., 

```ts
'$mod+([+=])': () => {
	zoom = Math.min(zoom + 0.0625, 3);
},
``` 

Or would you prefer to have two separate shortcuts?

Reply:
Oddly that regex one doesn't work for me on the english keyboard. I'll push the double one for now :+1: